### PR TITLE
Add .NET samples

### PR DESCRIPTION
**Pull request by cecilphillip-stripe**

### CODE REVIEW

**Comment by charliegerard-stripe** [comment]
Hey @cecilphillip-stripe! 👋 I'm having some trouble running this sample. I should be able to just run `dotnet run Program.cs`, right?

**Comment by cecilphillip-stripe** [comment]
@cecilphillip-stripe hhmm I'll run it again today and see what happens. `dotnet run` in the root directory should be all you need.

**Comment by cecilphillip-stripe** [comment]
@charliegerard-stripe added the missing Readme files. HTH

**Comment by charliegerard-stripe** [comment]
Oh weird, I got it to work now but the issue was that I was missing the `.env` file 🤦‍♀️ . It's weird cause it wasn't loading any UI at all whereas in other cases, when I forgot the file, there was still some of the UI showing and an error message either server-side or client-side 🤔

File: using-webhooks/server/python/requirements.txt [hunk 1]
```diff
@@ -7,8 +7,5 @@ itsdangerous==1.1.0
 Jinja2==2.11.3
 MarkupSafe==1.1.1
 python-dotenv==0.10.3
-requests==2.22.0
-stripe==2.29.3
```
Comment (charliegerard-stripe) [inline using-webhooks/server/python/requirements.txt:11]: Was this change intentional? If we remove the `stripe` package here, isn't it going to break the Python sample?

**Review by charliegerard-stripe — COMMENTED** [review]
Just left a comment to clarify something otherwise it's almost ready to merge! 🥳
